Runtimes/Core/runtime/CMakeLists.txt

```
@@ -99,7 +99,8 @@ target_compile_definitions(swiftRuntime
     $<$<BOOL:${SwiftCore_HAS_DLSYM}>:-DSWIFT_STDLIB_HAS_DLSYM>
     $<$<BOOL:${SwiftCore_ENABLE_FILESYSTEM_SUPPORT}>:-DSWIFT_STDLIB_HAS_FILESYSTEM>
     $<$<AND:$<BOOL:${SwiftCore_ENABLE_FILESYSTEM_SUPPORT}>,$<COMPILE_LANGUAGE:C,CXX>>:-DSWIFT_ARCH="${SwiftCore_ARCH_SUBDIR}">
-    $<$<AND:$<BOOL:${SwiftCore_ENABLE_FILESYSTEM_SUPPORT}>,$<COMPILE_LANGUAGE:C,CXX>>:-DSWIFT_LIB_SUBDIR="${SwiftCore_PLATFORM_SUBDIR}">)
+    $<$<AND:$<BOOL:${SwiftCore_ENABLE_FILESYSTEM_SUPPORT}>,$<COMPILE_LANGUAGE:C,CXX>>:-DSWIFT_LIB_SUBDIR="${SwiftCore_PLATFORM_SUBDIR}">
+    $<$<BOOL:${SwiftCore_ENABLE_BACKTRACE_REPORTING}>:-DSWIFT_STDLIB_SUPPORTS_BACKTRACE_REPORTING>)
```

(Optional) we may consider moving the closing parenthesis on a new line, so to make subsequent diffs more straightforward in their intent, e.g.

```suggestion
    $<$<BOOL:${SwiftCore_ENABLE_BACKTRACE_REPORTING}>:-DSWIFT_STDLIB_SUPPORTS_BACKTRACE_REPORTING>
    )
```

---

I guess this is a style question. I've seen it go either way and don't really have a personal preference anymore, so long as it's consistent within the project. @compnerd, what are your thoughts on putting the closing parenthesis on a newline?

---

It does have some benefits, especially when merging during conflict resolution where it's easy to accidentally end up with multiple close parens if they're on the same line as the last item.

---

I agree that consistency is super important. I do prefer the parenthesis on the same line as that is the style that CMake uses as a project and it happens to line up with C/C++ as well.

---

> ... style that CMake uses as a project...

CMake usually puts the closing paren on the newline: 

```
  bindexplib.cxx
  )
```
https://github.com/Kitware/CMake/blob/cf44774fd0ae57054ccad234dd762f6de2ac36a0/Source/CMakeLists.txt#L782-L783


```
    ${CMake_HAIKU_INCLUDE_DIRS}
  )
```
https://github.com/Kitware/CMake/blob/cf44774fd0ae57054ccad234dd762f6de2ac36a0/Source/CMakeLists.txt#L791-L792

---

If we decide that we want closing parentheses on a list on a new line, I can put up a separate PR to go through the project and do that. I think it's out of scope for this PR though.